mlir/include/mlir/Dialect/Rock/Tuning/RockTuning.h

```diff
@@ -73,6 +74,11 @@ LogicalResult tuningTableLookup(TuningTable *perfTable, ModuleOp &mod,
 LogicalResult tuningTableLookupByKey(TuningTable *perfTable,
                                      SmallVectorImpl<char> &out);
 
+bool isSplitKRequested(ModuleOp &mod, StringRef &perfConfig);
```

Comment:
You don't need to take `mod` by reference here.

Reply:
done